### ISSUE TITLE
Alternative model

### ISSUE THREAD

**Issue by ghost** [jomjol/neural-network-autotrain-digital-counter#3]
Hi,

i had problems with the digit model on a electrical meter. So I looked deeper in the code and saw, the models will be trained often with the history and stored model. This makes a much overfitted model.  Because of the shuffling every train, the validation and train data mixed and the overfitting is not visible.

TLDR;

I made various changes and created a notebook with an small effnet-model trained with knowledge distillation and added more data for better results.

Look at [Colab-Notebook](https://colab.research.google.com/drive/1-bTkSoz64lmaRDSmiM_Vypfsiv_fT3Nc?usp=sharing)

If you interested in I can create a pull request, to integrate it. 

It's running on my eep-cam very well.

**Comment by jomjol** [comment]
Wow, that way beyond my knowledge and I will try to understand this - may take some time.

As I would like to be able to maintain this in the future a pull request would be difficult for the time beeing. But if you could make a branch out of this, so I can test and play arround would be great!

**Comment by ghost** [comment]
Yes i could add it in a Branch in my fork. I am not familiar with github actions. So ist could take a while.

**Comment by ghost** [comment]
I created a branch on my fork. With manual action runs (1h20min).
https://github.com/haverland/neural-network-autotrain-digital-counter/tree/Effnet-KDM

If you create a branch, I could create a pull request on it.

**Comment by jomjol** [comment]
Good Idea, I have created the branch `Improved-Training` for this pull request: https://github.com/jomjol/neural-network-autotrain-digital-counter/tree/Improved-Training
